modules/python/src2/typing_stubs_generation/predefined_types.py

```diff
@@ -54,8 +54,12 @@
                             doc="Required length is 2"),
     AliasTypeNode.sequence_("Size2f", PrimitiveTypeNode.float_(),
                             doc="Required length is 2"),
-    AliasTypeNode.sequence_("Scalar", PrimitiveTypeNode.float_(),
-                            doc="Required length is at most 4"),
+    AliasTypeNode.union_(
```

Comment:
(I'm sorry it is small points). It seems indent are missing. If possible, cloud you format like "Mat" block ?

https://github.com/opencv/opencv/blob/9bb01e799f9f4ac97d7f40bba2c897dfa8350f63/modules/python/src2/typing_stubs_generation/predefined_types.py#L46-L51